OSDOCS-17045-2: CQA 1 for MACH-4 Cluster API for Compute Machines (#114216)
openshift/openshift-docs: modules/deploying-capi-machines-via-mapi-machine-sets.adoc

[id="deploying-capi-machines-via-mapi-machine-sets_{context}"]
= Deploying Cluster API compute machines by using a Machine API compute machine set

[role="_abstract"]
You can configure a Machine API compute machine set to deploy Cluster API compute machines.

> Use this process, you can test the Cluster API compute machine creation workflow without creating and scaling a Cluster API compute machine set.

Reviewer: This change doesn't make sense grammatically
```suggestion
With this process, you can test the Cluster API compute machine creation workflow without creating and scaling a Cluster API compute machine set.
```

Author: Done

  -n openshift-machine-api
----
+

> `<machine_set_name>`: Specifies the name of the Machine API compute machine set that you want to configure to deploy Cluster API compute machines.

Reviewer: This was valid as it was and the change is not. 

You could revert it, or (if you want to closely match the callout replacement doc), this is also valid:
```suggestion
Replace `<machine_set_name>` with the name of the Machine API compute machine set that you want to configure to deploy Cluster API compute machines.
```

Author: I just cannot understand why the "`<machine_set_name>`:" format has an issue. It also retains the "Specifies" wording. Reverted for the sake of moving this PR along.

Reviewer: ```
`<machine_set_name>`: Specifies the name of the Machine API compute machine set that you want to configure to deploy Cluster API compute machines.
```
is not any of the callout replacement methods we are using, it's sort of a mix of things. You might have meant it to be a DL? in that case, it needs another `:`.

  name: <machine_set_name>
  [...]
spec:
  authoritativeAPI: MachineAPI
  [...]
  template:
    [...]
    spec:
      authoritativeAPI: ClusterAPI
status:
  authoritativeAPI: MachineAPI
  [...]
----
+
where:
+
`spec.authoritativeAPI`:: Specifies the unconverted value for the Machine API compute machine set. Do not change the value in this part of the specification.
`spec.template.spec.authoritativeAPI`:: Specifies what `ClusterAPI` to configure the compute machine set to deploy Cluster API compute machines.

> `status.authoritativeAPI`:: Specifies the current value for the Machine API compute machine set. Do not change the value in this part of the specification.

Reviewer: Would revert the line breaks to keep the changes smaller. Also, the edit to former callout 2 changes the meaning in a way that matters. 

Would do this:

```suggestion
`spec.authoritativeAPI`::
Specifies the unconverted value for the Machine API compute machine set. 
Do not change the value in this part of the specification.
`spec.template.spec.authoritativeAPI`:: 
Specifies the authoritative API for the machine set. 
To configure the compute machine set to deploy Cluster API compute machines, set this value to `ClusterAPI`.
`status.authoritativeAPI`:: 
Specifies the current value for the Machine API compute machine set. 
Do not change the value in this part of the specification.
```

Author: Done
